pkg/storage/tpr/storage_interface_test.go

```diff
@@ -140,7 +141,9 @@ func TestCreateWithNoNamespace(t *testing.T) {
 	inputServiceBroker := &sc.ServiceBroker{
 		ObjectMeta: metav1.ObjectMeta{Name: name},
 		Spec: sc.ServiceBrokerSpec{
-			URL: "http://my-awesome-broker.io",
+			URL:            "http://my-awesome-broker.io",
+			RelistBehavior: "Duration",
+			RelistDuration: &metav1.Duration{15 * time.Minute},
```

Comment:
What are we trying to check there by setting those fields? If we want to make sure that serialization and persistence works correctly, you should set values different from default.